src/layerstack.stanza

```diff
@@ -576,49 +579,106 @@ doc:\<DOC>
        copper-35um
        soldermask
      ]
+    ```
   
-  Example 2: 4-layer with two adjacent prepreg layers
+  Example 2: 4-layer stack with three adjacent prepreg layers, such as "JLC04161H-7628B"
   ```
-    val copper-35um = Copper(0.035)
-    val prepreg = FR4(0.1, FR4-Material)
-    val copper-17_5um = Copper(0.0175)
-    val core = FR4(1.265, FR4-Material)
-    val stack = make-layer-stack("4-layer with two adjacent prepreg layers", top-layers) where :
+    val prepreg2 = FR4(0.2, FR4-Material)
+    val prepreg3 = FR4(0.3, FR4-Material)
+    make-layer-stack("4-layer stack with three adjacent prepreg layers", top-layers) where :
       val top-layers =  [
-        [copper-35um prepreg]
-        prepreg
+        [copper-35um [prepreg prepreg2 prepreg3]]
         [copper-17_5um core]
       ]
   ```
   The LayerStack created has these layers:
+    ```
+     [  copper-35um
+       prepreg ; three adjacent prepreg layers
+       prepreg2
+       prepreg3
+       copper-17_5um
+       core
+       copper-17_5um
+       prepreg3
+       prepreg2
+       prepreg
+       copper-35um
+     ]
+    ```
+
+  Example 3: 6-layer stack with even number of layers, such as "JLC06201H-3313A"
```

Comment:
For examples 1, 2, 3 - let's please use the `@snippet` marker and related `@snip-note` marker syntax here for the docstrings.

See here for more details:
https://jit-x.slack.com/docs/TKJEB34LS/F05RW135Y0J